examples/cerebro/channel_ops.go

```diff
@@ -204,15 +204,16 @@ func (o *Operator) handleResizeChannel(args []string) {
 		return
 	}
 
-	chainRPC, err := o.getChainRPC(network.ChainID)
+	chainRPC, err := o.getChainRPC(blockchain.ID)
 	if err != nil {
 		fmt.Printf("Failed to get RPC for chain %s: %s\n", chainID.String(), err.Error())
 		return
 	}
 
+	walletAddress := common.HexToAddress(o.config.Wallet.PublicKey().Address().String())
```

Comment:
I think we should add a direct `.Address()` getter for the wallet type in `sign` pkg. Obviously, not in this PR, but a "TODO" can be added there now.